src/content/it/2020/resource-hints.md

```diff
@@ -0,0 +1,366 @@
+
+{{ figure_markup(
+  image="resource-hint-adoption-onservice-worker-pages.png",
+  caption="Adozione delle resource hint nelle pagine di Service Worker.",
+  description="Un grafico a barre del tasso di adozione delle resource hint nelle pagine di Service Worker suddiviso per tipo di hint e fattore di form. Per desktop, il 29% delle pagine usa la resource hint `dns-prefetch`, il 47% usa `preload`, il 34% usa `preconnect`, il 10% usa `prefetch` e meno dell'1% usa `prerender`. Il cellulare è simile, tranne per il fatto che `dns-prefetch` ha un utilizzo del 30% (1% in più) e `preconnect` ha il 34% (13% in meno).",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTYAbLxN40s6mNR1jo0XDe_V4siN8TAsx2mryMp5IQmlJ-9O9eJxYROz7Rw6ozyFP6hlIZHxxh95GqX/pubchart?oid=252958553&format=interactive",
+  sheets_gid="691299508",
+  sql_file="adoption_service_workers.sql"
+) }}
+
+Per `preload` su desktop abbiamo un eccezionale tasso di adozione del 47% e `prefetch` un tasso di adozione del 10%. In entrambi i casi i dati sono molto più alti rispetto all'adozione media senza un Service Worker.
+
+Come accennato in precedenza, il [PRPL Pattern](https://addyosmani.com/blog/the-prpl-pattern/) giocherà un ruolo significativo nei prossimi anni nel modo in cui combineremo le resource hints con la strategia di caching del Service Worker.
+
+## Futuro
+
+Immergiamoci in un paio di hints sperimentali. Molto vicino al rilascio abbiamo i hint prioritari(Priority hints), che vengono attivamente sperimentati nella comunità web. Abbiamo anche i 103 Early Hints in HTTP/2, che è ancora all'inizio e ci sono alcuni giocatori come [Chrome e Fastly che collaborano per i prossimi test di prova](https://www.fastly.com/blog/beyond-server-push-experimenting-with-the-103-early-hints-status-code).
+
+### Hints prioritari
+
+[Priority hints](https://developers.google.com/web/updates/2019/02/priority-hints) sono un'API per esprimere la priorità di fetch di una risorsa: alta, bassa o automatica. Possono essere utilizzati per ridurre la priorità delle immagini (ad esempio all'interno di un Carousel), ridefinire la priorità degli script e persino aiutare a ridurre la priorità dei fetch.
+
+Questo nuovo hint può essere utilizzato come tag HTML o modificando la priorità delle richieste di fetch tramite l'opzione `importance`, che assume gli stessi valori dell'attributo HTML.
+
+```html
+<link rel="preload" href="/js/script.js" as="script" importance="low">
+
+<img src="/img/in_view_but_not_important.svg" importance="low" alt="I'm not important!">
```

Comment:
You removed the comments from here but there

```suggestion
<!-- Vogliamo avviare un recupero anticipato di una risorsa, ma anche ridimensionarla -->
<link rel="preload" href="/js/script.js" as="script" importance="low">

<!-- Un'immagine che il browser assegna priorità "Alta", ma in realtà non la vogliamo. -->
<img src="/img/in_view_but_not_important.svg" importance="low" alt="I'm not important!">
```

Reply:
@chefleo did you see this? Hidden under a "Load more..." so you might have missed it.